### DISCUSSION TITLE
Open terminal outside chroot in X

### DISCUSSION THREAD

**Discussion by saltedcoffii** [dnschneid/crouton#4560] [Q&A]
Does anybody know if there's a way to open a terminal into the ChromeOS bash shell (outside the chroot) while within X in the crouton chroot? I'd love any ideas/help.

**Comment by ShipmasterKyle** [comment]
Can't you just open a new tab?

**Reply by saltedcoffii** [reply]
Sorry I meant like when in a DE through start<de>, e.g. `startunity` or `startxfce4`. Like is there a way to open an `libvte` or `xterm` terminal outside the chroot.

**Comment by ghost** [comment]
The only option close to this would be to use `xiwi` and start just the terminal. You could also use `sudo enter-chroot` and it will give you terminal access inside of crosh.
